packages/lit-html/src/directives/cache.ts

```diff
@@ -71,7 +77,6 @@ class CacheDirective extends Directive {
           clearPart(containerPart);
           insertPart(containerPart, undefined, cachedPart);
           setCommittedValue(containerPart, [cachedPart]);
-          cachedPart.setConnected(true);
```

Comment:
This is removed because `insertPart` handles it now?

Reply:
Right